### PR TITLE
update manifests folder

### PR DESCRIPTION
**Pull request by israel-morales**
After testing with v14 of the API,  we discovered a few issues with the example manifests files

The goal for this PR is to have a fully working deployment (currently based off version 14), so that a user can run with the example and avoid troubleshooting errors

1.  include a full set of appgate crds (00-crd.yaml, version 14) 
2.  fix appgate operator deployment, as it seems the binary is looking for a specific configmap name (02-appgate-operator.yaml)
3.  fix role to include watching ALL appgate crds and basic configmap/secret objects (01-role.yaml) 

any feedback is welcome

thank you!

### CODE REVIEW

**Comment by lepiolet** [comment]
Thanks @israel-morales !!! It looks good.
I will take a look at this ASAP.

File: manifests/00-crd.yaml [hunk 1]
```diff
@@ -180,3 +179,291 @@ spec:
...
+    kind: ClientConnection
+    plural: clientconnections
+    shortNames:
+    - cli
+    singular: clientconnection
+  scope: Namespaced
+  versions:
+  - name: v1
+    served: true
+    storage: true
```
Comment (lepiolet) [inline manifests/00-crd.yaml:469]: new line missing

File: manifests/01-role.yaml [hunk 2]
```diff
@@ -4,8 +4,30 @@ metadata:
...
+  - adminmfasettingss
+  - trustedcertificates
+  - criteriascriptss
+  - devicescripts
+  - entitlementscripts
+  - appliancecustomizations
+  - globalsettingss
+  - clientconnections
   verbs: ["get", "watch", "list"]
+- apiGroups: [""]
```
Comment (lepiolet) [inline manifests/01-role.yaml:28]: ahh yes, we were missing the configmap and the secrets role :D

File: manifests/02-appgate-operator.yaml [hunk 3]
```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
         - name: appgate-operator
-          image: atuindev/appgate-operator:latest
+          image: ghcr.io/appgate/sdp-operator:v14
```
Comment (lepiolet) [inline manifests/02-appgate-operator.yaml:17]: 👍🏾

File: manifests/02-appgate-operator.yaml [hunk 4]
```diff
@@ -31,46 +31,46 @@ spec:
             - name: APPGATE_OPERATOR_HOST
               valueFrom:
                 configMapKeyRef:
-                  name: appgate-operator-conf
+                  name: appgate-sdp-configmap
```
Comment (lepiolet) [inline manifests/02-appgate-operator.yaml:34]: your naming is definitively better but could you update this as well https://github.com/appgate/sdp-operator/blob/master/manifests/01-config.yaml#L4 please.

Also maybe the references to that configmap in the README file.

**Review by lepiolet — CHANGES_REQUESTED** [review]
Looks good, just some minor changes requested. Thanks.

**Comment by lepiolet** [comment]
@thomascellerier @israel-morales I have reviewed the PR, it looks good just some minor changes requested.

File: manifests/02-appgate-operator.yaml (hunk 4 above)
Comment (israel-morales) [reply manifests/02-appgate-operator.yaml:34]: updated https://github.com/appgate/sdp-operator/blob/master/manifests/01-config.yaml#L4 and the README:

https://github.com/appgate/sdp-operator/pull/114/commits/e9c6f555bf1d1c871f766f148e8a81a4b9142c0e

File: manifests/00-crd.yaml (hunk 1 above)
Comment (israel-morales) [reply manifests/00-crd.yaml]: fixed:
https://github.com/appgate/sdp-operator/pull/114/commits/a1a4ac6b0ffded4360e53a4d9c4e5cf1945b1678

**Review by lepiolet — APPROVED** [review]
Thanks!

**Pull request merged**